### PR TITLE
PLUGIN-545 MAX/MIN support string

### PR DESCRIPTION
**Pull request by ardian4**
Jira Ticket: https://cdap.atlassian.net/browse/PLUGIN-545

### CODE REVIEW

File: src/main/java/io/cdap/plugin/aggregator/function/CompareFunction.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
```
Comment (CuriousVini) [inline src/main/java/io/cdap/plugin/aggregator/function/CompareFunction.java]: 2021 :)

**Review by CuriousVini — COMMENTED** [review]
could you please add unit tests for this cases to be handled?

**Comment by ardian4** [comment]
> could you please add unit tests for this cases to be handled?

PR updated

**Review by CuriousVini — APPROVED** [review]
lgtm. Please squash the commits

**Comment by bajram-adapt** [comment]
Squashed

**Review by CuriousVini — APPROVED** [review]

**Pull request merged**